app/views/releases/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="container">
+  <h1>Release</h1>
+  <hr/>
+  <h3><%= @release.title %> <i> (<%= @release.released_on.strftime('%B %d, %Y') %>) </i></h3>
+  <h5><%= @release.subtitle %></h5>
+  <p><%= @release.body %></p>
+  <%= link_to 'Back', releases_path, :class => "btn btn-primary btn-sm" %>
+  <%= link_to 'Edit', edit_release_path(@release), :class => "btn btn-danger btn-sm" %>
+</div>
```

Comment:
Could you, please,  add some space between these two buttons?

Reply:
Fixed the Edit button problem. Thanks for your help!
<img width="1280" alt="Screen Shot 2022-01-14 at 2 23 22 PM" src="https://user-images.githubusercontent.com/81119399/149594934-78ef3932-8273-4926-8d7f-6e0886932ca0.png">